ReactWindows/ReactNative.Shared/UIManager/BaseViewManager.cs

```diff
@@ -58,6 +67,7 @@ public void SetOpacity(TFrameworkElement view, double opacity)
         [ReactProp("overflow")]
         public void SetOverflow(TFrameworkElement view, string overflow)
         {
+#if WINDOWS_UWP
             if (overflow == "hidden")
```

Comment:
I think it would be better for react to fail explicitly when you try to set a native prop that doesn't exist than to have poorly documented no-op behavior for props that aren't implemented yet.

I actually think that React Native is designed to no-op already if a prop doesn't exist, which is significantly more efficient than actually having to extract the string from the JSON payload when it is set.

We should consider adding a strict-mode in the prop setter implementation to throw when someone tries to set a prop that doesn't exist on native.